spec/lib/evm_database_spec.rb

```diff
@@ -14,10 +14,31 @@
     end
   end
 
-  context "#seed_primordial" do
-    it "populates seeds" do
-      described_class::PRIMORDIAL_CLASSES.each { |klass| expect(klass.constantize).to receive(:seed) }
-      described_class.seed_primordial
+  context ".seed" do
+    def expect_proper_counts
+      expect(MiqDatabase.count).to eq(1)
+      expect(MiqRegion.count).to eq(1)
+      expect(MiqEnterprise.count).to eq(1)
+      expect(Zone.count).to eq(1)
+      expect(MiqServer.count).to eq(1)
+      expect(ServerRole.count).to eq(23)
+      expect(Tenant.count).to eq(1)
+      expect(MiqProductFeature.count).to eq(950)
```

Comment:
Was going to ask about this one; I did it mostly for myself. I also didn't like this as it would be by far the slowest spec in the entire suite. I'll gladly remove it.